.dockerignore

```diff
@@ -0,0 +1,35 @@
+.git
+# Ignore data, log & temp directories
+action_logs/
+!cache/
+cache/**
+!cache/img
+cache/img/**
+!cache/img/signatures
+cache/img/signatures/**
+!cache/img/signatures/static
+!cache/img/signatures/static/*
+tmp/
+
+# Ignore installation-related files
+install/lock
+
+# Ignore PHP-related files & directories
+vendor/
+
+# Ignore app's instance configuration
+config/**
+!config/.gitkeep
+
+# Custom files that should not be comitted into the base code
+language/**/rules.definitions.custom.lang
```

Comment:
This file should not be committed into the codebase, yes, however its existence is necessary for showing custom rules. I haven't tested this, but I think this entry makes it impossible to use custom rules definition? I'll try to verify this later.

Reply:
I have removed this line from `.dockerignore`